exercises/perfect-numbers/generator/generator.jl

```diff
@@ -0,0 +1,23 @@
+# canonical version = 1.3.0
```

Comment:
See above about the fixed version.

This file should be in [`generators/`](https://github.com/exercism/julia/tree/master/generators) instead.